src/io/inputs.jl

```diff
@@ -139,8 +98,8 @@ function _nc_dim_labels(
     sites = "reef_siteid" in keys(nc_file.vars) ? _site_labels(nc_file) : 1:size(data, 2)
 
     try
-        # This will be an issue if two or more dimensions have the same number of elements
-        # as the number of sites, but so far it hasn't happened...
+        # This will be an issue if number of elements is equal the number of sites, two or
+        # more dimensions but so far that hasn't happened...
```

Comment:
This sentence no longer makes sense.

```suggestion
        # This will be an issue if the number of elements for two or more dimensions have 
        # the same number of elements, but so far it hasn't happened...
```